cpp/ql/lib/semmle/code/cpp/Type.qll

```diff
@@ -1695,7 +1717,7 @@ class TemplateParameter extends UserType {
  * void foo(const Container<Elem> &value) { }
  * ```
  */
-class TemplateTemplateParameter extends TemplateParameter {
+class TemplateTemplateParameter extends TypeTemplateParameter {
```

Comment:
Based on the description in the change note I thought that type template parameters and template template parameters where to different groups and wouldn't have expected `TemplateTemplateParameter` to extend `TypeTemplateParameter`?

Reply:
I could pull that apart, but that will have quite some fallout in other places, and possibly for customers. So, I prefer to leave this as-is. Another observation is that template template parameters will still refer to (templated) types, so from that perspective it makes sense to have this layering, I think.

Reply:
I see, thanks. Let's keep it like this then :)